src/renderers/shared/reconciler/ReactReconciler.js

```diff
@@ -40,13 +41,15 @@ var ReactReconciler = {
     transaction,
     nativeParent,
     nativeContainerInfo,
-    context
+    context,
+		nodesToReuse
```

Comment:
I started at first with `nodeToReuse`, but I changed them all (I think!) to `nodesToReuse` after I realized that argument would sometimes be a single value and would at other times need to be an array. I chose plural everywhere to match the naming of `props.children`, which is plural but also a variable that can represent a single value or an array. I don't have strong feelings about this, and if you'll like me to change it, I'm happy to do so.